service.go

```diff
@@ -663,13 +663,14 @@ func (client *Client) ListServicesWithTier(tier string, variables *PayloadVariab
 	return &q.Account.Services, nil
 }
 
-func (client *Client) UpdateService(input ServiceUpdateInput) (*Service, error) {
+func (client *Client) UpdateService(input ServiceUpdater) (*Service, error) {
```

Comment:
There's a disadvantage to using `any` which is that it isn't obvious what would be accepted here as input without looking at the source code, and it would also ruin autocomplete. We have an unnecessary `default` case here that can be avoided by using the `ServiceUpdater` interface. The random functions can easily be auto-generated and are public so they will not be visible to end users.